### DISCUSSION TITLE
Bitclout ad feature brainstorming?

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#71] [Medium CIPs]
I think we should not be against the advertising industry. Yes, Twitter and Facebook use us as products and earn advertising income. Well, wouldn't it be nice if we could get our share of these advertising revenues? For example, there is an advertisement banner on the right of my Profile page and there is an NFT on this banner. Why would it be bad if someone buys this NFT thanks to the ad in my profile and I get my profit from this sale?

Or I suggested another Creator on my Profile page. And someone followed this account. Why shouldn't I profit from this follow? These examples can be diversified. But I don't find it right to avoid advertising.

**Comment by tijno** [comment]
Ads are a node specific decision / some may want to show them others wont.

Not sure we need a CIP to discuss this, unless you have a more specific proposal?

**Comment by sull** [comment]
Ads are not really something to consider on the protocol level or any of the core functionality. It is a frontend thing and if payouts are to be done in the native currency ($clout) or creator coins perhaps, this could be done as a second layer and/or 3rd party service and need not be integrated any deeper than that. 

Also, any post (including NFTs) could be used as ads, albeit with a different approach to traditional ads.

> Or I suggested another Creator on my Profile page. And someone followed this account

This could still be done in the traditional tracking method way, just on backend there would be a payment handler using the bitclout APIs to send $clout as ad or affiliate related revenue payouts.
